package.json

```diff
@@ -15,8 +15,11 @@
   "scripts": {
     "lint": "eslint doc.js test/*",
     "pretest": "npm run lint && npm run build",
-    "test": "bash test.sh",
-    "build": "uglifyjs doc.js -o doc.min.js --compress drop_console=true --mangle --screw-ie8"
+    "test": "mocha-chrome --reporter spec 'http://localhost:8888/test/docTest.js.html'",
```

Comment:
O `test` era um script bash porque dependia da subida do servidor de testes primeiro. Se for alterar isso, é necessário fazer com que o servidor suba de alguma outra forma; pode ser por um script aqui do `package.json` mesmo. E também é necessário alterar o arquivo `.travis.yml`, porque atualmente ele está configurado para chamar diretamente o script `test.sh`.